tools/cpp_build/libtorch/CMakeLists.txt

```diff
@@ -201,3 +201,21 @@ INSTALL(TARGETS test_jit
   RUNTIME DESTINATION "${TORCH_INSTALL_BIN_DIR}"
   LIBRARY DESTINATION "${TORCH_INSTALL_LIB_DIR}"
   ARCHIVE DESTINATION "${TORCH_INSTALL_LIB_DIR}")
+
+IF(NOT NO_CUDA)
+  SET(TORCH_TEST_SRCS ${TORCH_SRC_DIR}/csrc/cuda/test_rng.cpp)
+
+  ADD_EXECUTABLE(test_rng ${TORCH_TEST_SRCS})
+
+  TARGET_LINK_LIBRARIES(test_rng torch)
+
+  TARGET_INCLUDE_DIRECTORIES(test_rng PUBLIC
+    "${TORCH_SRC_DIR}/../"
+    "${TORCH_SRC_DIR}/../aten/src/ATen/utils/catch/single_include"
+    "${COMMON_INCLUDES}")
+
+  INSTALL(TARGETS test_rng
+    RUNTIME DESTINATION "${TORCH_INSTALL_BIN_DIR}"
+    LIBRARY DESTINATION "${TORCH_INSTALL_LIB_DIR}"
+    ARCHIVE DESTINATION "${TORCH_INSTALL_LIB_DIR}")
```

Comment:
That's much better, but:
1. Can you please move `test_rng.cpp` out of `torch/csrc` and into `test/cpp`? I know that's where `test_jit.cpp` is, but that's for legacy reasons. We don't want to keep tests right next to our code.
2. Please remove the `INSTALL` line. No one wants to install the tests.

Reply:
Will is going to move it to `aten/src/ATen/test`